docs/src/main/paradox/_template/projectSpecificFooter.st

```diff
@@ -0,0 +1,4 @@
+<script type="text/javascript" src="$page.base$assets/js/warnOldVersion.js"></script>
+<script type="text/javascript">//<![CDATA[
+jQuery(function(jq){initOldVersionWarnings(jq, '$page.properties.("project.version")$', '$page.properties.("project.url")$')});
+//]]></script>
```

Comment:
Should these eventually move to https://github.com/akka/akka-paradox or a separate paradox plugin?

Reply:
Yes, I get tired of copying it everywhere. But there is Akka's more advanced version handling in the template, isn't it?